src-docs/src/assets/version-selector.js

```diff
@@ -127,7 +127,7 @@ const tpl = `
     </style>
     <a id="root" role="button" aria-labelledby="selected" aria-controls="dropdown" tabindex="0">
       <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" role="img" 
-       width="1em" height="1em" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24">
+       width="1em" height="1em" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24" aria-hidden="true">
```

Comment:
Context for reviewers: `OuiIcon` already adds the `aria-hidden` attribute given certain criteria. This icon would meet that criteria. Not sure why this wasn't there in the first place